app/tsconfig.json

```diff
@@ -12,13 +13,13 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     // "sourceMap": true,                     /* Generates corresponding '.map' file. */
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "./",
+    //    "outDir": "./",
     /* Redirect output structure to the directory. */
     // "rootDir": "./",                       /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     // "tsBuildInfoFile": "./",               /* Specify file to store incremental compilation information */
     // "removeComments": true,                /* Do not emit comments to output. */
-    "noEmit": true,
+    //    "noEmit": true,
```

Comment:
Why did you comment out this? Assuming we _don't_ want to emit outputs

Reply:
> Seems to be faster 🚀 , but still taking ~`Time: 28979ms` on initial load?
> 
> What are your typescript IDE settings @enjeyw ?

Yeah the initial load won't be much faster, because it still needs to build the vendor package (node modules) the first time  round - the improvement is that those modules are no longer built on later changes, so ongoing deving should be quicker. It's still not as quick as i'd like mind u

Reply:
> Why did you comment out this? Assuming we don't want to emit outputs

Nah the webpack now uses typescript directly to compile, so we do need the inputs now. Try adding noemit back in - it'll fail!